packages/closer/pages/settings/index.tsx

```
@@ -30,20 +31,24 @@
-const SettingsPage = () => {
+const SettingsPage = ({
+  volunteerConfig,
+}: {
+  volunteerConfig: VolunteerConfig;
+}) => {
   const { APP_NAME } = useConfig();
 
+  const skillsOptions =
+    (volunteerConfig?.skills && volunteerConfig?.skills.split(',')) || [];
+  const dietOptions =
+    (volunteerConfig?.diet && volunteerConfig?.diet?.split(',')) || [];
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)